vpr/src/pack/cluster.cpp

```diff
@@ -1371,7 +1401,12 @@ static bool cleanup_pb(t_pb* pb) {
 }
 
 /**
- * Try pack molecule into current cluster
+ * Performs legality checks to see whether the selected molecule can be
+ * packed into the current cluster. The legality checks are related to
+ * floorplanning, pin feasibility, and routing (if detailed route
+ * checking is enabled). The routine returns BLK_PASSED if the molecule
+ * can be packed in the cluster. Otherwise, it returns the appropriate
+ * failed pack status based on which legality check the molecule failed.
```

Comment:
Add this: If the block passes, the routine commits it to the current cluster and updates the appropriate data structures